Tests/VauxTests/VauxTests.swift

```diff
@@ -292,6 +355,32 @@ final class VauxTests: XCTestCase {
       XCTFail(error.localizedDescription)
     }
   }
+    
+  func testAttributes() {
+    func buildPage() -> HTML {
+        html {
+            div {
+                "Custom tag text goes here"
+                }.attr("key", "value")
```

Comment:
Can you please fix the indentation of the closing bracket to be aligned with the `div` here? Also, this project uses 2 spaces for tabs, so please adjust all of your code accordingly.

Reply:
Additionally, feel free to test out some of the other strongly-typed attributes here, such as `class` or `id`.